packages/create-app/src/cli.ts

```diff
@@ -132,6 +140,7 @@ export async function cli(args: string[] = process.argv): Promise<void> {
   const osdkPackage: string = await promptOsdkPackage(parsed);
   const osdkRegistryUrl: string = await promptOsdkRegistryUrl(parsed);
   const corsProxy: boolean = await promptCorsProxy(parsed);
+  const scopes: string[] | undefined = await promptScopes(parsed);
```

Comment:
We want to allow the `scopes` option to be entirely optional to support backward compatability

Reply:
Instead of it being optional should we instead have a default value e.g `["api:read-data", "api:write-data"]`

Reply:
I think this goes back to the discussion of whether we should be exposing end users to the concept of `scopes` if they aren't "interested". By allowing this to be set as optional, we can avoid generating the templates for users who do not require the `scopes` to avoid confusion. I'm okay with providing default as well but I don't think it strongly aligns with the other places where we do apply these `scopes` display like the instructions on the documentation